### ISSUE TITLE
order-in-components -> allow dumb computed

### ISSUE THREAD

**Issue by michalsnik** [netguru/eslint-plugin-netguru-ember#36]
Currently computed property without any dependents results in wrong order warning.
Many existing projects however uses this method instead of setting this property inside `init` hook.

Basically code below should not cause errors/warnings:
```
export default Component.extend({
  role: "sloth",
  levelOfHappiness: computed(function() {}),
  actions: {}
});
```

**Comment by michalsnik** [comment]
Done in `v1.6.3`
